Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -283,6 +283,37 @@
   components:
   - type: Unremoveable
 
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesNorthStar
+  name: gloves of the north star
+  description: These gloves allow you to punch incredibly fast
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/northstar.rsi
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/northstar.rsi
+  - type: MeleeWeapon
+    attackRate: 4
+    damage:
+      types:
+       Blunt: 7.5
+    soundHit:
+      collection: Punch
+    animation: WeaponArcFist
+  - type: Fiber
+    fiberMaterial: fibers-leather
+    fiberColor: fibers-blue
+  - type: FingerprintMask
+  - type: MeleeSpeech
+  - type: ActivatableUI 
+    key: enum.MeleeSpeechUiKey.Key
+    closeOnHandDeselect: false
```

Comment:
Activate the UI with right click or alt click instead. If you use the "E" key, it has a mispredict and equips first, then unequips and opens the UI menu.